tests/playground.html

```diff
@@ -137,7 +142,24 @@
 
 function getToolboxElement() {
   var match = location.search.match(/toolbox=([^&]+)/);
-  return document.getElementById('toolbox-' + (match ? match[1] : 'categories'));
+  // Default to the basic toolbox with categories and untyped variables,
+  // but override that if the toolbox type is set in the URL.
+  var toolboxSuffix = (match ? match[1] : 'categories');
```

Comment:
'categories' should be 'toolbox-categories', right?

Reply:
No, because it's a suffix.  It's added to 'toolbox-' on line 150.